ProfileView/src/index.ts

```diff
@@ -66,24 +64,43 @@ export default class CjaasProfileWidget extends LitElement {
   @property({ type: String, attribute: "base-stream-url" }) baseStreamURL:
     | string
     | undefined = undefined;
-
-  // timeline properties
-  @property({ type: Array }) timelineItems: TimelineItem[] = [];
-  @property() filter: string | undefined;
-  @property({ reflect: true }) pagination = "$top=15";
   @property({ type: Number }) limit = 5;
-  @property({ reflect: true }) timelineType:
-    | "journey"
-    | "livestream"
-    | "journey-and-stream" = "livestream";
-
+  @internalProperty() liveStream = false;
+  @internalProperty() events: CustomerEvent[] = [];
+  @internalProperty() newestEvents: Array<CustomerEvent> = [];
   @internalProperty() eventSource: EventSource | null = null;
   @internalProperty() showTimelineSpinner = false;
   @internalProperty() errorMessage = "";
-
   @internalProperty() profile: any;
   @internalProperty() showSpinner = false;
 
+  async lifecycleTasks() {
+    const data = await this.getExistingEvents();
+    this.events = data.events;
+    this.showSpinner = false;
+    this.requestUpdate();
+    this.subscribeToStream();
+  }
+
+  async firstUpdated(changedProperties: PropertyValues) {
+    super.firstUpdated(changedProperties);
+    await this.lifecycleTasks();
+
+    // Preserve for responsive display development
```

Comment:
Are we not worrying about responsiveness for now?

Reply:
I added a higher level CSS  solution so I could get rid of the more complex resize logic.